SQL/SQL.sublime-syntax

```diff
@@ -9,9 +9,8 @@ file_extensions:
   - dml
 
 first_line_match: |-
-  (?xi:
-    ^ \s* -- .*? -\*- .*? \bsql\b .*? -\*-  # editorconfig
-  )
+  (?xi)
```

Comment:
please revert this change

Reply:
Ok. Thought it's consistent with the rest of the formatting I did...

Reply:
You should never use this style for variables, as it'll have unintended consequences when using them.

Reply:
You're right, for variables it's probably not the best. But `first_line_match` isn't really a variable, right?
Now that you're saying that, I'll have to check whether this is an issue in ElixirSyntax. Maybe I was lucky and didn't run into issues with `(?x)` in variables...

Reply:
Sure, then let me rephrase this: Explicit is better than implicit.